docs/website/docs/dlt-ecosystem/verified-sources/personio.md

```diff
@@ -0,0 +1,189 @@
+
+1. Sign in to your Personio account, and ensure that your user account has API access rights.
+1. Navigate to Settings > Integrations > API credentials.
+1. Click on "Generate new credentials."
+1. Assign necessary permissions to credentials, i.e. read access.
+
+### Initialize the verified source
+
+To get started with your data pipeline, follow these steps:
+
+1. Enter the following command:
+
+   ```bash
+   dlt init personio duckdb
+   ```
+
+   [This command](../../reference/command-line-interface) will initialize
+   [the pipeline example](https://github.com/dlt-hub/verified-sources/blob/master/sources/personio_pipeline.py)
+   with Personio as the [source](../../general-usage/source) and [duckdb](../destinations/duckdb.md)
+   as the [destination](../destinations).
+
+1. If you'd like to use a different destination, simply replace `duckdb` with the name of your
+   preferred [destination](../destinations).
+
+1. After running this command, a new directory will be created with the necessary files and
+   configuration settings to get started.
+
+For more information, read the
+[Walkthrough: Add a verified source.](../../walkthroughs/add-a-verified-source)
+
+### Add credentials
+
+1. In the `.dlt` folder, there's a file called `secrets.toml`. It's where you store sensitive
+   information securely, like access tokens. Keep this file safe. Here's its format for service
+   account authentication:
+
+   ```toml
+   # Put your secret values and credentials here
+   # Note: Do not share this file and do not push it to GitHub!
+   [sources.personio]
+   client_id = "papi-*****" # please set me up!
+   client_secret = "papi-*****" # please set me up!
+   ```
+
+1. Replace the value of `client_id` and `client_secret` with the one that
+   [you copied above](#grab-credentials). This will ensure that your data-verified source can access
+   your Personio API resources securely.
+
+1. Next, follow the instructions in [Destinations](../destinations/duckdb) to add credentials for
+   your chosen destination. This will ensure that your data is properly routed to its final
+   destination.
+
+For more information, read the [General Usage: Credentials.](../../general-usage/credentials)
+
+## Run the pipeline
+
+1. Before running the pipeline, ensure that you have installed all the necessary dependencies by
+   running the command:
+   ```bash
+   pip install -r requirements.txt
+   ```
+1. You're now ready to run the pipeline! To get started, run the following command:
+   ```bash
+   python personio_pipeline.py
+   ```
+1. Once the pipeline has finished running, you can verify that everything loaded correctly by using
+   the following command:
+   ```bash
+   dlt pipeline <pipeline_name> show
+   ```
+   For example, the `pipeline_name` for the above pipeline example is `personio`, you may also use
+   any custom name instead.
+
+For more information, read the [Walkthrough: Run a pipeline.](../../walkthroughs/run-a-pipeline)
```

Comment:
we periodically reorganize docs structure, so we don't have Walkthrough section anymore, we need to replace it with something neutral, at least:
For more information, read [Run a pipeline.](../../walkthroughs/run-a-pipeline)

Reply:
done.